docs/operations-guide/scaling-metabase.md

```diff
@@ -0,0 +1,142 @@
+
+The application database stores all of your questions, dashboards, collections, permissions, and other data related to the Metabase application. We recommend you use an external database (like PostgreSQL or MySQL) to manage your application database. You can also use a managed relational database, like AWS RDS, which will auto-scale for your needs.
+
+### Upgrade to the latest version of Metabase
+
+If you haven't already, we recommend you update to the latest Metabase version to get the most recent performance improvements.
+
+### Only ask for the data you need
+
+In general, you should only query the data you need. If you set up a dashboard that you'll be viewing daily, you can reduce load times by limiting the number of records your queries return.
+
+If you have many users running queries that return a lot of records, it won't matter that Metabase is fast: the users will get their data only as fast as your database(s) can return the requested records.
+
+Take advantage of Metabase's data exploration tools to learn about your data and preview records in tables so you can dial in on only the records you need to answer your question.
+
+Be especially mindful when querying data across time or space, as you can filter out a lot of unnecessary data by restricting your question to a shorter timespan or smaller area. Do you really need to see data years back, or in all geohashes?
+
+### Cache your queries
+
+You can [configure caching](../administration-guide/14-caching.md) on questions to store their results. Metabase will show users the timestamp for the results, and users can manually refresh the results if they want to rerun the query. Caching is suitable for results that do not update frequently.
+
+### Look for bottlenecks
+
+Metabase's Enterprise Edition offers [auditing tools](../enterprise-guide/audit.md) for you to monitor the usage and performance of your application. You can, for example, see how many questions are being asked, by whom, and how long the questions took to run, which can help identify any bottlenecks that need attention.
+
+### Keep dashboards to 7 questions or fewer
+
+Sometimes people go overboard with dashboards, loading them up with 50 questions or more. When a dashboard with 50 questions loads, it sends 50 simultaneous requests asking for data. And depending on the size of that database and the number of tables in that database, it can be quite some time before those records return to answer all of those questions.
+
+7 is, of course, an arbitrary number; you can create dashboards where adding a lot of questions makes sense. In general, though, encourage your users to keep their dashboards focused. Dashboards are meant to tell a story about your data, and you can tell a good story with just a handful of questions. If you find that one of your dashboards has accumulated a lot of questions, consider breaking it up into multiple dashboards that each focus on a set of related questions.
```

Comment:
Given this an arbitrary, I'd rather not be specific. This will all depend on the backend. If your DB executes all 50 queries in under 1 second, you're great.